feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -184,19 +188,35 @@ fun PreviewScreen(
                         )
                     }
 
-                    TestingButton(
+                    Row(
                         modifier = Modifier
-                            .testTag("ToggleCaptureMode")
                             .align(Alignment.TopEnd)
                             .padding(12.dp),
-                        onClick = { viewModel.toggleCaptureMode() },
-                        text = stringResource(
-                            when (previewUiState.currentCameraSettings.captureMode) {
-                                CaptureMode.SINGLE_STREAM -> R.string.capture_mode_single_stream
-                                CaptureMode.MULTI_STREAM -> R.string.capture_mode_multi_stream
-                            }
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        TestingButton(
+                            modifier = Modifier
+                                .testTag("ToggleCaptureMode"),
+                            onClick = { viewModel.toggleCaptureMode() },
+                            text = stringResource(
+                                when (previewUiState.currentCameraSettings.captureMode) {
+                                    CaptureMode.SINGLE_STREAM -> R.string.capture_mode_single_stream
+                                    CaptureMode.MULTI_STREAM -> R.string.capture_mode_multi_stream
+                                }
+                            )
                         )
-                    )
+                        if (previewUiState.currentCameraSettings.supportedStabilizationMode !=
+                            SupportedStabilizationMode.UNSUPPORTED &&
```

Comment:
Shouldn't it be the case that if the current stabilization mode is `UNSUPPORTED` then the stabilization mode can't be set to `ON`? If so, then we can get rid of the check for `UNSUPPORTED`, we should just go with whatever the stabilization mode is set to.